x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -62,6 +62,7 @@ export function Cytoscape({
   // Trigger a custom "data" event when data changes
   useEffect(() => {
     if (cy) {
+      // cy.remove(cy.nodes());
```

Comment:
is this not needed? cytoscape will dedupe automatically by id and id only? if so, we can remove this line completely right?

Reply:
correct, i just forgot to remove this line